src/EnergyPlus/WaterCoils.cc

```diff
@@ -2472,9 +2470,14 @@ void SizeWaterCoil(EnergyPlusData &state, int const CoilNum)
             CoolingWaterDesWaterInletTempSizer sizerCWDesWaterInTemp;
             sizerCWDesWaterInTemp.initializeWithinEP(state, CompType, CompName, bPRINT, RoutineName);
             state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp = sizerCWDesWaterInTemp.size(state, TempSize, ErrorsFound);
-
-            if (state.dataWaterCoils->WaterCoil(CoilNum).UseDesignInletWaterTemp) {
-                state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp = state.dataWaterCoils->WaterCoil(CoilNum).DesignInletWaterTemp;
+            if (state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp > state.dataSize->DataDesOutletAirTemp) {
+                state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp = state.dataSize->DataDesOutletAirTemp - 5.0;
+                ShowWarningError(state, "Invalid design inlet water temperature for " + CompType + " = " + CompName);
+                ShowContinueError(state,
+                                  format("...design inlet water temperature = {:.3R} C", state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp));
```

Comment:
You are reporting here state.dataSize->DataDesOutletAirTemp - 5.0, I think you wanted to report the original inlet water temperature?

Reply:
You are correct. I will fill fix that.